aibolit/utils/ast.py

```diff
@@ -186,9 +194,12 @@ def get_type(self, node: int) -> ASTNodeType:
     def get_method_invoked_name(self, invocation_node: int) -> MethodInvocationParams:
         assert(self.get_type(invocation_node) == ASTNodeType.METHOD_INVOCATION)
         # first two STRING nodes represent object and method names
-        object_name, method_name = islice(self.children_with_type(invocation_node, ASTNodeType.STRING), 2)
-        return MethodInvocationParams(self.get_attr(object_name, 'string'),
-                                      self.get_attr(method_name, 'string'))
+        children = list(self.children_with_type(invocation_node, ASTNodeType.STRING))
+        if len(children) == 1:
+            return MethodInvocationParams('inline method', self.get_attr(children[0], 'string'))
```

Comment:
I don't think there is a `inline method` term in Java. We should better make first line empty if we call for a local method.